### PR TITLE
Document new Chef Backend 1.4 configurables

### PR DESCRIPTION
**Pull request by stevendanna**
Signed-off-by: Steven Danna <steve@chef.io>

### CODE REVIEW

File: chef_master/source/install_server_ha.rst [hunk 1]
```diff
@@ -620,6 +620,9 @@ Chef HA backend leader management service settings
 * ``leaderl.etcd_pool.ibrowse_options`` Internal options to affect how
   requests to etcd are made (see
   https://github.com/cmullaparthi/ibrowse/blob/master/doc/ibrowse.html).
+* ``leaderl.epmd_monitor.check_interval`` How often (in milliseconds)
+  to check that leaderl is registered with the Erlang Port Mapping
+  Deamon (epmd).  ``60000`` by default.
```
Comment (srenatus) [inline chef_master/source/install_server_ha.rst]: `s/ea/ae/`

**Review by srenatus — APPROVED** [review]
🚀 (without the ~typo~ transpo)

File: chef_master/source/install_server_ha.rst (hunk 1 above)
Comment (stevendanna) [reply chef_master/source/install_server_ha.rst]: :D

**Review by srenatus — APPROVED** [review]
🐰

**Comment by srenatus** [comment]
@stevendanna Just noticed this doesn't _say 1.4_ anywhere. Can we assume that readers assume the docs describe the latest release?

**Comment by stevendanna** [comment]
@srenatus I was going to put "since 1.4+" in the descriptions but noticed we don't have that for any other configuration options. I'm happy to add it though.  Perhaps @davidwrede and crew can weigh in on what they would prefer?

**Review by afiune — APPROVED** [review]

**Comment by brewn** [comment]
@stevendanna I like the idea of adding a version note - I can put one in before this gets merged.

**Pull request merged**